enqueue-function.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * This script is for a Lambda that ingests the  data from a POST request from API Gateway,
+ * logs some of the information to dynamo db and sends the body of the request with a
+ * unique id on to two different SQS Queues.
+ *
+ */
+const AWS = require('aws-sdk');
+
+//Logs the request information to Dynamo DB
+function logNewReq(unique_id, request_info) {
+    let client_id = request_info.user_id;
+
+    console.log(
+        'logging new request ',
+        { request_id, request_id },
+        typeof request_info,
+    );
+
+    const dbParams = {
+        TableName: 'gateway_requests',
+        Item: {
+            client_id,
+            unique_id,
+            name: request_info.name,
+            status: 'enqueued',
+            created_at: new Date().toString(),
+            updated_at: new Date().toString(),
+            location: request_info.location,
+            target: request_info.target,
+            recipient_email: request_ifno.email,
+            set_live: request_info.set_live,
+        },
+    };
+
+    const docClient = new AWS.DynamoDB.DocumentClient();
+
+    return docClient.put(dbParams).promise();
+}
+
+/**
+ * Handles our Lambda event
+ * @param {*} event
+ * @param {*} context
+ */
+exports.handler = async (event, context) => {
+    console.log('Incoming event: ', event);
+
+    const SQS = new AWS.SQS();
+
+    let queue1_name = 'register-request-queue.fifo';
+    const queue1_url = (
+        await SQS.getQueueUrl({ QueueName: queue1_name }).promise()
+    ).QueueUrl;
+
+    const enqueue_params = {
+        QueueUrl: queue1_url,
+        MessageGroupId: 'prod',
+        MessageBody: event,
+    };
+
+    var unique_id = context.awsRequestId;
```

Comment:
```suggestion
    const unique_id = context.awsRequestId;
```
`var` has function scope which can cause unexpected behavior. Best to use `let` (mutable) or in this case, `const` because this value will not change